webpack.config.prod.js

```diff
@@ -52,7 +54,15 @@ module.exports = Object.assign(prodConfig, {
     new webpack.optimize.AggressiveMergingPlugin(),
     new MiniCssExtractPlugin({
       filename: "styles-[name]-[contenthash].css"
-    })
+    }),
+    ...process.env.IS_NOT_CI ? [new SentryWebpackPlugin({
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+      org : process.env.ORG_NAME,
+      project : process.env.PROJECT_NAME,
+      release : process.env.SENTRY_RELEASE,
```

Comment:
A few things here:
- You should open a ticket with devops to set these values
- Could you namespace them all under `SENTRY_*`?
- Currently the release version comes from `settings.py`, there's no environment variable for that and we don't currently have any automation around setting this var either. So I think some work needs to go into pulling that  out into a version file or otherwise getting it extracted somehow during the heroku build. The release bot expects it to live in `settings.py` though so the latter might be best for the time being.

Reply:
I have addressed the first two issues However I am not sure I understand you last point. Is there any related document or commit I could look in to

Reply:
I've put up a fix to the release script here: https://github.com/mitodl/release-script/pull/358, that will allow you to move the version into a `VERSION` file and then replace the current definition of `VERSION` in `settings.py` with a few lines of code that read that file's contents. Then do the same in `webpack.config.prod.js` and use that for the value of `release`.

Reply:
Ok, that PR is approved, but I'm going to hold off on merging it until after this merged, let me know when i can review this again.

Reply:
Yeah, i had addressed the feedback a while back but forgot tor re-request the review. Sorry for the delay

Reply:
@Wassaf-Shahzad I'm not seeing those changes here, specifically the usage of a `VERSION` file.